### ISSUE TITLE
Default pl_batch_shrink requires batch to be >= 2

### ISSUE THREAD

**Issue by futscdav** [NVlabs/stylegan2-ada-pytorch#24]
The batch size is int divided by `loss.pl_batch_shrink` on [this line](https://github.com/NVlabs/stylegan2-ada-pytorch/blob/ecfea65b463fd4f59b39eb8bdbaec2c70a6644eb/training/loss.py#L79), which leads to an error, since new batch size will be 0. As train.py specifies that batch must be at least 1, the value should probably change to 1 if batch == 1 or train.py should reflect that minimum batch size setting is 2.

**Comment by MISSEY** [comment]
@nurpax
Having the similar issue, trying to understand the code by putting batch size =1 , which leads to an error 

`tensor([], device='cuda:0', size=(0, 18, 512), grad_fn=<CopySlices>)
cannot reshape tensor of 0 elements into shape [0, 1, -1, 1, 1] because the unspecified dimension size -1 can be any value and is ambiguous`

at line https://github.com/NVlabs/stylegan2-ada-pytorch/blob/ecfea65b463fd4f59b39eb8bdbaec2c70a6644eb/training/networks.py#L56

Could you please suggest what should I do to fix the issue ? 

Thank you
